packages/postgres-database/scripts/erd/Dockerfile

```diff
@@ -16,15 +16,14 @@ RUN apt-get update \
 
 
 RUN --mount=type=cache,mode=0755,target=/root/.cache/pip \
-  pip install --upgrade \
+  pip --no-cache-dir install --upgrade \
```

Comment:
are you aware that putting no cache dir here kind of defeats the purpose of caching from the host?
```--mount=type=cache,mode=0755,target=/root/.cache/pip``` vs ```--no-cache-dir```?

Reply:
Now i am :-) thx